pkg/build/api/types.go

```diff
@@ -197,7 +197,7 @@ type DockerBuildStrategy struct {
 // STIBuildStrategy defines input parameters specific to an STI build.
 type STIBuildStrategy struct {
 	// Image is the image used to execute the build.
-	Image string `json:"image,omitempty"`
+	From kapi.ObjectReference `json:"from,omitempty"`
```

Comment:
@bparees we were discussing this with @yin and I do like From more than Image ;-) What are your thoughts?